pulpcore/pulpcore/app/viewsets/repository.py

```diff
@@ -239,11 +239,27 @@ class PublisherViewSet(CreateReadAsyncUpdateDestroyNamedModelViewset):
     filter_class = PublisherFilter
 
 
+class PublicationFilter(filterset.FilterSet):
+    """
+    A base Publication filter that always excludes incomplete Publications.
+    """
+
+    class Meta:
+        model = Publication
+        fields = ('created', 'publisher', 'repository_version', 'complete')
```

Comment:
I'm unsure whether the `complete` field should be listed here; unsure what the purpose of the `Meta.fields` actually is. More docs digging I shall....